cli/wallet/nep5.go

```diff
@@ -87,3 +112,95 @@ func getNEP5Balance(ctx *cli.Context) error {
 	}
 	return nil
 }
+
+func transferNEP5(ctx *cli.Context) error {
+	wall, err := openWallet(ctx.String("path"))
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	defer wall.Close()
+
+	addr := ctx.String("from")
+	from, err := address.StringToUint160(addr)
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("invalid `from` address: %v", err), 1)
+	}
+	acc := wall.GetAccount(from)
+	if acc == nil {
+		return cli.NewExitError(fmt.Errorf("can't find account for the address: %s", addr), 1)
+	}
+
+	tokenHash, err := util.Uint160DecodeStringLE(ctx.String("token"))
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("invalid token contract hash: %v", err), 1)
+	}
+
+	to, err := address.StringToUint160(ctx.String("to"))
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("invalid `to` address: %v", err), 1)
+	}
+
+	gctx, cancel := getGoContext(ctx)
+	defer cancel()
+
+	c, err := client.New(gctx, ctx.String("rpc"), client.Options{})
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	result, err := c.InvokeFunction(tokenHash.StringLE(), "decimals", []smartcontract.Parameter{})
```

Comment:
We probably want to have more wrappers in RPC client for standard NEP5 queries.

Reply:
Done!